org/teams.yaml

```diff
@@ -398,6 +398,10 @@ teams:
         - jwendell
         - ryantking
         - zhlsunshine
+      Release Manager - 1.12:
```

Comment:
Will there eventually be more than one?

This should probably say "Managers"

Reply:
I believe the other release manager will be @dgn. Just updated the PR